packages/react-reconciler/src/ReactFiberComponentStack.js

```diff
@@ -129,9 +132,12 @@ export function getOwnerStackByFiberInDev(workInProgress: Fiber): string {
       case SuspenseListComponent:
         info += describeBuiltInComponentFrame('SuspenseList');
         break;
+      case ActivityComponent:
+        info += describeBuiltInComponentFrame('Activity');
+        break;
       case ViewTransitionComponent:
         if (enableViewTransition) {
-          info += describeBuiltInComponentFrame('SuspenseList');
+          info += describeBuiltInComponentFrame('ViewTransition');
```

Comment:
@sebmarkbage I fixed this from https://github.com/facebook/react/pull/32034, and added some tests for the parent stack, but I'm struggling to find a way to cause these built ins to get hit. 

`SuspenseList` can hit it because of the prop validation, but how about `Activity` or `ViewTransition`? I'm sure it's something obvious.

Reply:
ViewTransition doesn't really have much validation or errors. Otherwise these don't trigger.

The ViewTransition events should probably execute in the context of this Fiber as the owner which would give that.

However, that's not implemented yet:

https://github.com/facebook/react/blob/main/packages/react-reconciler/src/ReactFiberWorkLoop.js#L838

There's also no error handling around these events which there should be.